src/components/Home.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import banners from "../data/featured-banners.json";
+import productCategories from "../data/product-categories.json";
+import FeaturedProducts from "./FeaturedProducts";
+
+const DivCategory = styled.div`
+  border: 1px solid;
+  min-width: 30vh;
+  background-color: #5472d3;
+`;
+
+const ImageInSlider = styled.img`
+  max-height: 100vh;
+  width: 100vh;
+`;
+
+function Categories() {
+  const categories = productCategories.results.map((obj) => {
+    return obj.data.name;
+  });
+  const [categoryItems, setCategoryitems] = useState(categories.slice(0, 3));
+  const [section, setSection] = React.useState(0);
+
+  const Category = ({ name }) => {
+    return (
+      <DivCategory key={name} className="category">
+        <p>{name}</p>
+      </DivCategory>
+    );
+  };
+
+  function Elements(props) {
```

Comment:
My last suggestion here as a good practice is move the Categories and Elements functions to a different files, to have the logic separated and in the case if you need to add more logic in the Categories function, the Home component doesn't need to increase in size. This is a very good clean code practice.

Reply:
Agree, makes sense, adjusted the Home page for this segregation.